src/write.c

```diff
@@ -320,59 +320,61 @@ void avifEncoderSetCodecSpecificOption(avifEncoder * encoder, const char * key,
     avifCodecSpecificOptionsSet(encoder->csOptions, key, value);
 }
 
-static void avifBackupSettings(avifEncoder * encoder)
+static void avifEncoderBackupSettings(avifEncoder * encoder)
```

Comment:
This function copies many members, one by one. Should we just copy the entire `avifEncoder` struct?

```
    *lastEncoder = *encoder;
```